other-docs/guides/getting-help-with-altis.md

```diff
@@ -1,50 +1,90 @@
+
+**CCs**
+
+You can CC additional relevant parties to the support request if desired. They will recieve carbon copies of the correspondance.
+
+## Support Ticket Types
+
+We break down support into different categories. Each has its own language and processes, but it should be somewhat intuitive. Let's go over the types.
+
+![Altis Dashboard Create Support Ticket page, Question selected in Type dropdown menu](./assets/altis-support-type.png)
 
 We currently have four types of support. Below is a description of each and how they are different from each other.
 
 ### Incident
 
-Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an Incident, and requires investigation.
-
-From the support side, Incidents get a rapid response. If the site is down, this is an Incident, with a priority of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
+Incidents are unplanned events that negatively impact service, or the quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
 
-Incidents focus on immediate attention, with an emphasis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
+From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
+If the site is down for example, select the 'Incident' type, with a priority of ‘Urgent’.
 
 ### Problem
 
-Problems are the cause of, or potential cause of Incidents. Incidents revolve around something unexpected. If a site is down, that’s an Incident, but the Problem is a known cause of an issue you wish to track, and coordinate help with. Problems typically take longer to resolve, as an appropriate solution is thought out and worked on.
+Problems are the cause of, or can be the potential cause of future Incidents. If you're encountering a problem which could cause potential Incidents, let us know!
 
 ### Task 
 
-There’s an array of tasks that might you need help with. Typically, this might mean provisioning of infrastructure to accommodate some need, it might be providing access to some tool or service used by the team, or it could be to run an operation that requires a cloud engineer. Tasks should be well described.
+There’s an array of tasks that might you need help with. Typically, this might mean provisioning of infrastructure to accommodate some need, it might be providing access to some tool or service used by the team, or it could be to run an operation that requires a cloud engineer. 
```

Comment:
Looks like there are some words that are swapped around. Didn't catch it in the first pass.
```suggestion
There’s an array of tasks that you might need help with. Typically, this might mean provisioning of infrastructure to accommodate some need, it might be providing access to some tool or service used by the team, or it could be to run an operation that requires a cloud engineer. 
```